src/integ-test/java/com/mathworks/ci/RunMATLABCommandInteg.java

```diff
@@ -0,0 +1,156 @@
+package com.mathworks.ci;
+
+import com.gargoylesoftware.htmlunit.*;
+import com.gargoylesoftware.htmlunit.html.*;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import hudson.tasks.Builder;
+import org.junit.*;
+import hudson.EnvVars;
+import hudson.matrix.Axis;
+import hudson.matrix.AxisList;
+import hudson.matrix.Combination;
+import hudson.matrix.MatrixBuild;
+import hudson.matrix.MatrixProject;
+import hudson.matrix.MatrixRun;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import hudson.tasks.Builder;
+
+
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.*;
+
+
+public class RunMATLABCommandInteg {
+
+    private FreeStyleProject project;
+    private UseMatlabVersionBuildWrapper buildWrapper;
+    private RunMatlabCommandBuilder scriptBuilder;
+
+    @Rule
+    public JenkinsRule jenkins = new JenkinsRule();
+
+    @Before
+    public void testSetup() throws IOException {
+
+        this.project = jenkins.createFreeStyleProject();
+        this.scriptBuilder = new RunMatlabCommandBuilder();
+        this.buildWrapper = new UseMatlabVersionBuildWrapper();
+    }
+
+    @After
+    public void testTearDown() {
+        this.project = null;
+        this.scriptBuilder = null;
+    }
+
+    private String getMatlabroot() throws URISyntaxException {
+        String ML_version = TestData.getPropValues("matlab.version");
```

Comment:
Is the version of MATLAB specified here  also taken as input for MATLAB windows/linux root path?